apps/communications/dialer/test/marionette/lib/views/dialer_options/views.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+/* global module */
+//view needs its accessor
+var DialerOptionsAccessor = require('./accessors');
+//views will instatiate the other views or accessor?, here should need the one for call log
+
+//var CallLogAccessor = 
+
+//var appRoot = require('app-root-path');
+// TODO Change the path once requireFromApp becomes its own module
+//var fromApp = require(appRoot +
+//  '/shared/test/integration/require_from_app');
+var appRoot = require('app-root-path');
+// TODO Change the path once requireFromApp becomes its own module
+var fromApp = require(appRoot +
+  '/shared/test/integration/require_from_app');
+
+
+function DialerOptionsView(client) {
+  this.client = client;
+
+  this.accessors = new DialerOptionsAccessor(client);
+
+}
+
+DialerOptionsView.prototype = {
+ 
+  goToCallLog: function() {
+    var callLogTab = this.accessors.callLogButton;
+    callLogTab.tap();
+
+    var CallLogView = require('../call_log/views.js');
+    var callLogView = new CallLogView(this.client);
+    callLogView.accessors.waitToAppear();
```

Comment:
`waitToAppear()` is not needed if the constructor of `CallLogView` already makes sure it has appeared. Then we can simplify this function by:
```js
return new CallLogView(this.client);
```